sql-plugin/src/main/scala/com/nvidia/spark/rapids/AlluxioCfgUtils.scala

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright (c) 2022, NVIDIA CORPORATION.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.nvidia.spark.rapids
+
+import org.apache.spark.sql.execution.datasources.FileFormat
+import org.apache.spark.sql.execution.datasources.parquet.ParquetFileFormat
+
+/*
+ * Utils for the alluxio configurations.
+ * If alluxio feature is disabled, we can tolerate the client jar is absent.
+ * Use this pattern to solve the `NoClassDefFoundError` in `MortgageSparkSuite`:
+ * if (AlluxioCfgUtils.isAlluxioEnabled) { AlluxioCfgUtils does not import any alluxio class
```

Comment:
this comment seems odd because there is not AlluxioCfgUtils.isAlluxioEnabled function in here.  I'm assuming you mean choose the appropriate enable* function below and call that first before ever calling anything from alluxio client, correct?

Reply:
You are right. I updated the comments.